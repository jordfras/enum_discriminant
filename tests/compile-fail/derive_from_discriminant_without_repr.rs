use enum_discriminant::FromDiscriminant;

#[derive(FromDiscriminant)]
enum MyEnum {
    Variant,
}

fn main() {}
