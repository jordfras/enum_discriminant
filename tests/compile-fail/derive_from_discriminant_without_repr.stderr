error: When deriving FromDiscriminant on an enum, you also need to specify representation type with #[repr()] or #[discriminant()]
 --> tests/compile-fail/derive_from_discriminant_without_repr.rs:4:1
  |
4 | / enum MyEnum {
5 | |     Variant,
6 | | }
  | |_^
